src/app/components/header/date-now/date-now.component.less

```diff
@@ -0,0 +1,13 @@
+.date-now { 
+    font-size: 16px;
+    font-weight: bold;
+    color: rgba(0,0,0,.5);
+    cursor: pointer;
+    border: none;
+    background-color: #E0E0E0;
+    padding: 7px 15px;
+    border-radius: 50px;
+    &:focus {
+        outline: none;
+    }
+}
```

Comment:
I think we need follow style guide 
Example
![image](https://user-images.githubusercontent.com/20131942/103076437-684b1100-45d6-11eb-99d4-9ac1f252e618.png)